src/transformers/utils/hub.py

```diff
@@ -1081,6 +1093,27 @@ def extract_info_from_url(url):
     return {"repo": cache_repo, "revision": revision, "filename": filename}
 
 
+def create_and_tag_model_card(repo_id, tags=None):
+    """
+    Creates a dummy model card and tags it.
+    """
+    model_card = None
+
+    try:
+        # Check if the model card is present on the remote repo
+        model_card = ModelCard.load(repo_id)
+    except EntryNotFoundError:
+        # Otherwise create a simple model card from template
+        card_data = ModelCardData(language="en", tags=[])
+        model_card = ModelCard.from_template(card_data)
+
+    if model_card is not None and tags is not None:
+        for model_tag in tags:
+            model_card.data.tags.append(model_tag)
```

Comment:
```suggestion
        for model_tag in tags:
            if model_tag not in model_card.data.tags:
                model_card.data.tags.append(model_tag)
```
(just in case tag already exists on the Hub)